book/website/communication/presentations/public-speaking-tips.md

```diff
@@ -0,0 +1,91 @@
+# Practical Techniques for Effective Public Speaking (or "A Guide to Impactful Presentations")
+
+Public speaking, like most other skills, is one you can never truly master, but only strive to refine. 
+
+Below are some practical tips that can radically sharpen your public speaking skills. 
+Whether you’re prepping for a talk, a webinar, or even a casual presentation, these gems might just transform how you approach your next speaking opportunity.
```

Comment:
I suggest "preparing" instead of "prepping" here.

Reply:
Great suggestion! 💯

Reply:
Thank you!